dist/typings.d.ts

```diff
@@ -32,7 +32,10 @@ export declare type ConnectionSettingType = {
 export declare type CustomerSupportType = {
     view?: 'customer-support';
 };
-export declare type OpenServicesConfigsOptions = ConfigsOptions & (ServicesListType | ServiceConnectionType | ConnectionSettingType | CustomerSupportType);
+export declare type MyAccountPageType = {
+    view?: 'authorized-applications' | 'change-language' | 'email-preferences' | 'delete-account' | 'update-email' | 'update-password';
```

Comment:
Isn't it better to make the the same as [line 68](https://github.com/moneytree/mt-link-javascript-sdk/pull/79/files#diff-fea2eda0d3d1195024f3ecf8b2df0e68d9717448281c60df0fbceb56132ec2ddR68?)

Reply:
Fixed.